README.md

```diff
@@ -212,6 +213,7 @@ presented separately here.
 
 <!-- 75 -->
 [Parcel JavaScript Transformer]: https://github.com/parcel-bundler/parcel
+[Calcit]: https://github.com/calcit-lang/calcit_runner.rs
```

Comment:
unstable grouping method...?

Reply:
If you have an idea for changing the grouping method, go ahead and suggest one. 👍🏻